internal/services/mysql/mysql_flexible_server_resource_test.go

```diff
@@ -1359,3 +1388,20 @@ resource "azurerm_mysql_flexible_server" "test" {
 }
 `, r.template(data), acceptance.WriteOnlyKeyVaultSecretTemplate(data, secret), data.RandomInteger, version)
 }
+
+func (r MySqlFlexibleServerResource) publicNetworkAccess(data acceptance.TestData, publicNetworkAccessEnabled string) string {
```

Comment:
NB: I made `publicNetworkAccessEnabled` a string so it could be `true`, `false` or `null`.

Reply:
This test now uses the respective Enum type.